client/src/components/BubbleMark.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
This file unused?

Reply:
No. Meant to remove it… but then removed the component that used it first and forgot to come back to remove it. 🤦